libminifi/src/RemoteProcessorGroupPort.cpp

```diff
@@ -150,6 +194,87 @@ void RemoteProcessorGroupPort::onTrigger(core::ProcessContext *context, core::Pr
   return;
 }
 
+void RemoteProcessorGroupPort::refreshRemoteSite2SiteInfo() {
+  if (this->host_.empty() || this->port_ == -1 || this->protocol_.empty())
+      return;
+
+  std::string fullUrl = this->protocol_ + this->host_ + ":" + std::to_string(this->port_) + "/nifi-api/controller/";
```

Comment:
Thanks for posting that @benqiu2016. I learned something new about the NiFI REST API :).

Are you planning to add support for default protocol ports?